packages/cli/src/constructs/__tests__/browser-check.spec.ts

```diff
@@ -10,20 +11,21 @@ describe('BrowserCheck', () => {
   it('should correctly load file dependencies', () => {
     Session.basePath = __dirname
     Session.availableRuntimes = runtimes
-    const bundle = BrowserCheck.bundle(path.join(__dirname, 'fixtures', 'browser-check', 'entrypoint.js'), '2022.10')
+    const getFilePath = (filename: string) => path.join(__dirname, 'fixtures', 'browser-check', filename)
+    const bundle = BrowserCheck.bundle(getFilePath('entrypoint.js'), '2022.10')
     delete Session.basePath
 
     expect(bundle).toEqual({
-      script: 'require("./dep1")\nrequire("./dep2")\n',
+      script: fs.readFileSync(getFilePath('entrypoint.js')).toString(),
```

Comment:
For some reason, this test was failing on Windows. For example, see https://github.com/checkly/checkly-cli/actions/runs/4322499095/jobs/7544988073.

It must be that reading the file on Windows doesn't give the same string as reading it on MacOS/Linux. I'm not sure that it's worth opening a bug and investigating further right now, though.

```
Error: expect(received).toEqual(expected) // deep equality

- Expected  - 4
+ Received  + 4

  Object {
    "dependencies": Array [
      Object {
-       "content": "module.exports = \"dependency 1\"
+       "content": "module.exports = \"dependency 1\"
  ",
        "path": "fixtures/browser-check/dep1.js",
      },
      Object {
-       "content": "module.exports = \"dependency 2\"
+       "content": "module.exports = \"dependency 2\"
  ",
        "path": "fixtures/browser-check/dep2.js",
      },
    ],
-   "script": "require(\"./dep1\")
- require(\"./dep2\")
+   "script": "require(\"./dep1\")
+ require(\"./dep2\")
  ",
    "scriptPath": "fixtures/browser-check/entrypoint.js",
  }

    at Object.<anonymous> (src/constructs/__tests__/browser-check.spec.ts:16:20)
```